src/main/java/by/itech/lab/supplier/service/TaxService.java

```diff
@@ -0,0 +1,12 @@
+package by.itech.lab.supplier.service;
+
+import by.itech.lab.supplier.dto.TaxDto;
+
+import java.util.List;
+
+public interface TaxService {
```

Comment:
We have BaseService. Do we need to implement it? In case not we could return optional for tax by state method.